pyproject.toml

```diff
@@ -16,6 +16,7 @@ classifiers=[
 dependencies = [
     "caikit[runtime-grpc,runtime-http]>=0.25.0,<0.26.0",
     "caikit-tgis-backend>=0.1.25,<0.2.0",
+    "caikit-tgis-backend>=0.1.27,<0.2.0",
```

Comment:
this seems to have become a duplicate line

Reply:
for emphasis?

Reply:
(fixed)